[2025-08-01T04:32:56.358+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-01T04:32:56.388+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parquet_in_minio.download_parquet scheduled__2024-09-01T00:00:00+00:00 [queued]>
[2025-08-01T04:32:56.395+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parquet_in_minio.download_parquet scheduled__2024-09-01T00:00:00+00:00 [queued]>
[2025-08-01T04:32:56.396+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2025-08-01T04:32:56.419+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_parquet> on 2024-09-01 00:00:00+00:00
[2025-08-01T04:32:56.426+0000] {standard_task_runner.py:63} INFO - Started process 1735 to run task
[2025-08-01T04:32:56.432+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'parquet_in_minio', 'download_parquet', 'scheduled__2024-09-01T00:00:00+00:00', '--job-id', '411', '--raw', '--subdir', 'DAGS_FOLDER/parquet_to_minio.py', '--cfg-path', '/tmp/tmpjhz3kdvq']
[2025-08-01T04:32:56.435+0000] {standard_task_runner.py:91} INFO - Job 411: Subtask download_parquet
[2025-08-01T04:32:56.451+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-01T04:32:56.486+0000] {task_command.py:426} INFO - Running <TaskInstance: parquet_in_minio.download_parquet scheduled__2024-09-01T00:00:00+00:00 [running]> on host 87b19f697adc
[2025-08-01T04:32:56.584+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='parquet_in_minio' AIRFLOW_CTX_TASK_ID='download_parquet' AIRFLOW_CTX_EXECUTION_DATE='2024-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-01T00:00:00+00:00'
[2025-08-01T04:32:56.585+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-01T04:32:56.599+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:320 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-08-01T04:32:57.330+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-01T04:32:57.330+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/parquet_to_minio.py", line 45, in download_parquet
    response.raise_for_status()
  File "/home/airflow/.local/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://d37ci6vzurychx.cloudfront.net/trip-data/green_tripdata_2024-08.parquet
[2025-08-01T04:32:57.343+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=parquet_in_minio, task_id=download_parquet, run_id=scheduled__2024-09-01T00:00:00+00:00, execution_date=20240901T000000, start_date=20250801T043256, end_date=20250801T043257
[2025-08-01T04:32:57.360+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 411 for task download_parquet (403 Client Error: Forbidden for url: https://d37ci6vzurychx.cloudfront.net/trip-data/green_tripdata_2024-08.parquet; 1735)
[2025-08-01T04:32:57.403+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-01T04:32:57.424+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-01T04:32:57.426+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
